views/js/runner/plugins/controls/progressbar/progressbar.js

```diff
@@ -175,10 +274,18 @@ define([
              * Update the progress bar
              */
             var update = function update (){
-                var progressData = progressUpdater(testRunner.getTestContext(), testRunner.getTestMap(), progressConfig);
-                if(progressData && $progressLabel && $progressControl){
-                    $progressLabel.text(progressData.label);
-                    $progressControl.progressbar('value', progressData.ratio);
+                var testContext = testRunner.getTestContext();
+                var testMap = testRunner.getTestMap();
+                var progressData = progressUpdater(testContext, testMap, progressConfig);
+                var item = mapHelper.getItemAt(testMap, testContext.itemPosition);
+                if (item && item.informational) {
+                    self.$element.hide();
```

Comment:
I have a concern about that: the progress bar will be hidden for all informational items... and for all customers...
You should rather apply this behavior depending on an option